### PR TITLE
[FIX] mail: prevent messaging menu mark as read from opening thread

### PR DESCRIPTION
**Pull request by seb-odoo**
task-2278759

### CODE REVIEW

**Comment by seb-odoo** [comment]
Extracted from https://github.com/odoo/odoo/pull/53143

**Comment by seb-odoo** [comment]
PS: test is currently relying on the assumption that a chat window would be open in the problematic case, except it is not the case no matter what because current behavior of preview is just to trigger up.

However in PR about open, I'll change this behavior and immediately call open instead of triggering up, then the test will make more sense. Still not good that open thread testing relies on chat window but this is a more general problem.

**Comment by alexkuhn** [comment]
> PS: test is currently relying on the assumption that a chat window would be open in the problematic case, except it is not the case no matter what because current behavior of preview is just to trigger up.
> 
> However in PR about open, I'll change this behavior and immediately call open instead of triggering up, then the test will make more sense. Still not good that open thread testing relies on chat window but this is a more general problem.

I would add task-id in the tests to be adapted. This PR will likely be merged soon and before the one you mentioned (is it the PR from task-2282238 [1]? If so, PR is not linked in the pad)


[1] https://www.odoo.com/web#id=2282238&action=3206&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720

**Comment by seb-odoo** [comment]
The test doesn't need to be adapted, it's just "useless" at asserting the thread is not open in current sate, the assertion will always pass. Once refactor this onclick to call open instead of using trigger the test will still pass, but with an assertion that can actually fail if there is an issue.

File: addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview_tests.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        '.o_ThreadNeedactionPreview_markAsRead',
+        "should have 1 mark as read button"
+    );
+
+    await afterNextRender(() =>
+        document.querySelector('.o_ThreadNeedactionPreview_markAsRead').click()
+    );
+    assert.verifySteps(
+        ['set_message_done'],
+        "should have set message done"
```
Comment (alexkuhn) [inline addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview_tests.js:87]: would rephrase "should have marked message as read", "set message done" is strange & too technical (the name of the python method is bad).

Comment (seb-odoo) [reply addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview_tests.js:87]: done

**Comment by alexkuhn** [comment]
@robodoo r+
